.pre-commit-config.yaml

```
@@ -42,7 +42,7 @@ repos:
     rev: v4.0.0-alpha.8
```

Let's give this a go:

```suggestion
    rev: v3.1.0
```